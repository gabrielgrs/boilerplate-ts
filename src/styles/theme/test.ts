import * as theme from './index'

describe('theme', () => {
  it('Theme should match snapshot', () => {
    expect(theme).toMatchSnapshot()
  })
})
