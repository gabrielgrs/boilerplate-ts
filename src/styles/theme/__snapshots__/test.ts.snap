// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme Theme should match snapshot 1`] = `
Object {
  "colors": Object {
    "primary": "#0f0f0f",
    "white": "#fff",
  },
  "sizes": Object {
    "xl": "8px",
    "xxs": "4px",
  },
}
`;
